lib/outputs/outputs.py

```diff
@@ -212,8 +223,12 @@ def fill_json_asff(finding_output, audit_info, finding):
         )
     ]
     finding_output.Compliance = Compliance(
-        Status=finding.status, RelatedRequirements=[finding.check_metadata.CheckType]
+        Status=finding.status + "ED",
```

Comment:
What is `ED`?

Reply:
Security Hub only accept either PASSED or FAILED, since we put in the checks PASS/FAIL we have to add that suffix.

Reply:
Please, include a comment with that, it's hard to identify what is happening.